apps/ex_wire/test/support/ex_wire/fake_kademlia_server.ex

```diff
@@ -0,0 +1,25 @@
+defmodule ExWire.FakeKademliaServer do
```

Comment:
We can create behaviour with kademlia functions and implement them here and in real kademlia module

Reply:
Yeah, though this is a genserver, so it doesn't work quite well with traditional behaviours since the functions are mostly `handle_call`, etc.